src/pages/components/paragraph.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Paragraph
+
+description: Use Paragraph to render text blocks with the appropriate Garden styling.
+package: '@zendeskgarden/react-typography'
+components:
+  - typography/src/elements/LG.tsx
+  - typography/src/elements/MD.tsx
+  - typography/src/elements/Paragraph.tsx
+  - typography/src/elements/SM.tsx
+---
+
+## How to use it
+
+### Default
+
+Use Paragraph to style and format `<p>` elements.
+
+import Default from '../../examples/typography/paragraph/Default.tsx';
+import DefaultCode from '!!raw-loader!../../examples/typography/paragraph/Default.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Size
+
+Paragraphs come in small, medium, and large. The default size is medium. Size
```

Comment:
Size controls the space between Paragraphs. The default size is medium, and you can choose small or large too. Use [typography](/components/typography) to choose an appropriate type scale.